src/state/collectibles/hooks.ts

```diff
@@ -23,6 +24,6 @@ export const useGetCollectibles = () => {
     isInitialized,
     isLoading,
     tokenIds: data,
-    nftsInWallet: Nfts.filter((nft) => identifiers.includes(nft.identifier)),
+    nftsInWallet: Nfts.pancake.filter((nft) => identifiers.includes(nft.identifier)),
```

Comment:
This gets it from the pancake collection. Perhaps we should rename this hook to `useGetPancakeCollection` or something.